Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Podverse/Podverse Mobile App](https://hosted.weblate.org/projects/podverse/podverse-rn/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/podverse/podverse-rn/horizontal-auto.svg)

blomusti committed:
Translated using Weblate (Turkish)

Currently translated at 7.1% (61 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/tr/

Commit:
Added translation using Weblate (Japanese)

Commit:
Translated using Weblate (Japanese)

Currently translated at 5.0% (43 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/ja/

d0dg3r committed:
Translated using Weblate (German)

Currently translated at 98.2% (843 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/de/

Atalanttore committed:
Translated using Weblate (German)

Currently translated at 98.8% (848 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/de/

jonycoo committed:
Translated using Weblate (German)

Currently translated at 98.8% (848 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/de/

Atalanttore committed:
Translated using Weblate (German)

Currently translated at 98.8% (848 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/de/

Commit:
Added translation using Weblate (Tamil)

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/ta/

Commit:
Translated using Weblate (Russian)

Currently translated at 77.0% (661 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 99.6% (855 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/ru/

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/es/

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/es/

fitojb commented:
I recommend enabling squashing commits by author on Weblate add-ons, or otherwise my changes will spam the repo’s history :/

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/es/

Commit:
Translated using Weblate (German)

Currently translated at 99.8% (857 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/de/

yurtpage committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/ru/

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/es/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (858 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/ta/

Commit:
Translated using Weblate (French)

Currently translated at 73.6% (632 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/fr/

Commit:
Translated using Weblate (Italian)

Currently translated at 33.7% (290 of 858 strings)

Translation: Podverse/Podverse Mobile App
Translate-URL: https://hosted.weblate.org/projects/podverse/podverse-rn/it/
